### ISSUE TITLE
about target domain images

### ISSUE THREAD

**Issue by renjie2001** [Mazin-Hnewa/MS-DAYOLO#11]
I want to know if the target domain images need augment during model training? If you can help me, I will thank you so much

**Comment by Mazin-Hnewa** [comment]
Yes, the target domain images are augmented with source domain images during training, but without ground-truth annotations. Only source images need to have ground-truth annotations.
